src/pandroid/app/src/main/java/com/panda3ds/pandroid/input/KeyName.java

```diff
@@ -0,0 +1,38 @@
+package com.panda3ds.pandroid.input;
+
+import com.panda3ds.pandroid.utils.Constants;
+
+public enum KeyName {
+    A(Constants.INPUT_KEY_A),
+    B(Constants.INPUT_KEY_B),
+    X(Constants.INPUT_KEY_X),
+    Y(Constants.INPUT_KEY_Y),
+    UP(Constants.INPUT_KEY_UP),
+    DOWN(Constants.INPUT_KEY_DOWN),
+    LEFT(Constants.INPUT_KEY_LEFT),
+    RIGHT(Constants.INPUT_KEY_RIGHT),
+    AXIS_LEFT,
+    AXIS_RIGHT,
+    AXIS_UP,
+    AXIS_DOWN,
+    START(Constants.INPUT_KEY_START),
+    SELECT(Constants.INPUT_KEY_SELECT),
+    L(Constants.INPUT_KEY_L),
+    R(Constants.INPUT_KEY_R),
+    NULL;
```

Comment:
Is this actually used anywhere?

Reply:
Nvm it is, might be best to use an invalid mask for it then

Reply:
i'm using in: `InputMap.java`, `InputMapPreferences`, `AlberInputListener`

Reply:
i'm replacing by invalid mask see the constructor

Reply:
-1 is not a proper invalid mask, it's all ones so if you were to eg to use it for checking if a button is pressed, then it would return true if any button is pressed.

Also the "NULL" enum and -1 are still separate entities

Reply:
the mask just to avoid if else, see: AlberInputListener

Reply:
...what?

Reply:
The `KeyName` is a `enum` for stores a 3ds input keys for use in java code, i'm using this for mapping keyboard/gamepad.
The enums allow stores some values
i'm using that enum for check keypressed, (i'm comaring enum names, see `InputMap.java`, and enum == enum), but for some keys in 3ds (L,R,UP,DOWN,LEFT,RIGHT,START, A/B/X/Y), It would be a waste to put a line for each of them because all need a same code, so i'm adding a keymask as a field in KeyName, the mask is only for this

Reply:
[AlberInputListener:32](https://github.com/GabrielBRDeveloper/Panda3DS-Android/blob/ui/src/pandroid/app/src/main/java/com/panda3ds/pandroid/app/game/AlberInputListener.java#L32)
[AlberInputListener:37](https://github.com/GabrielBRDeveloper/Panda3DS-Android/blob/ui/src/pandroid/app/src/main/java/com/panda3ds/pandroid/app/game/AlberInputListener.java#L37)